invenio_rdm_records/resources/serializers/bibtex/schema.py

```diff
@@ -227,9 +236,7 @@ def _fetch_fields_map(self, data):
             "doi": data.get("doi", None),
             "month": data.get("date_created", {}).get("month", None),
             "version": data.get("version", None),
-            "url": (lambda doi: None if doi is None else "https://doi.org/" + doi)(
-                data.get("doi", None)
-            ),
+            "url": data.get("url", None),
```

Comment:
if the result is `None`, is the url field added with empty value or missing?

Reply:
It is not added to the final string